x-pack/metricbeat/module/awsfargate/cloudformation.yml

```diff
@@ -63,7 +67,7 @@ Resources:
             - sh
             - -c
           Command:
-            - ./metricbeat setup && ./metricbeat modules disable system && ./metricbeat modules enable awsfargate && ./metricbeat -e -E cloud.id=$ELASTIC_CLOUD_ID -E cloud.auth=$ELASTIC_CLOUD_AUTH
+            - ./metricbeat modules disable system && ./metricbeat modules enable awsfargate && ./metricbeat -e -E cloud.id=$ELASTIC_CLOUD_ID -E cloud.auth=$ELASTIC_CLOUD_AUTH
```

Comment:
Wondering if we should setup the dashboard by default. I think it is something users would like it, wdyt?

Reply:
Yep! I still need to fix this. Plain `./metricbeat setup` doesn't work, I think it also needs `-E cloud.id` and `cloud.auth`. Will update it in next commit 👍

Reply:
👍🏼